cmd/argocd/commands/admin/backup.go

```diff
@@ -472,3 +512,18 @@ func updateTracking(bak, live *unstructured.Unstructured) {
 		}
 	}
 }
+
+// skipResource checks if a resource should be skipped based on its labels.
+func skipResource(bak *unstructured.Unstructured, skipResourcesWithLabels []string) bool {
+	for _, kv := range skipResourcesWithLabels {
+		parts := strings.SplitN(kv, "=", 2)
+		if len(parts) != 2 {
```

Comment:
I think it is fair to have a label without value, such as `--skip-resources-with-labels my-label/example.io` and exclude all resources that has the label. The backup process may not know all the possible values of that label.

Reply:
agreed